source-code/sdk-js/src/adapter-sveltekit/vite-plugin/transforms/refactoring.md

```diff
@@ -113,6 +113,144 @@ for markup block:
 
 # Flows
 
+## `Javascript` and `Svelte`
+
+There is some stuff that has to be done both in javascript AND every svelte file:
+
+Possible implementation:
+
+```ts
+function wrapAndAddParameters(
+	ast,
+	config,
+	wrappers: (
+		| "initHandleWrapper"
+		| "initRootLayoutServerLoadWrapper"
+		| "initServerLoadWrapper"
+		| "initActionWrapper"
+		| "initRequestHandlerWrapper"
+		| "initRootLayoutLoadWrapper"
+		| "initLoadWrapper"
+		| "initRootPageLoadWrapper"
+	)[],
+) {
+	wrappers.forEach((wrapper) => {
+		let identifier
+		let wrapperAst
+		switch (wrapper) {
+			case "initHandleWrapper":
+				identifiers = ["handle"]
+				wrapperAst = parse(`${wrapper}(/*options here*/).wrap`)
+				importFrom = "@inlang/sdk-js/adapter-sveltekit/server"
+				break
+			case "initServerLoadWrapper":
+			case "initRootLayoutServerLoadWrapper":
+				identifiers = ["load"]
+				wrapperAst = parse(`${wrapper}(/*options here*/).wrap`)
+				importFrom = "@inlang/sdk-js/adapter-sveltekit/server"
+				break
+			case "initRootLayoutLoadWrapper":
+			case "initRootPageLoadWrapper":
+			case "initLoadWrapper":
+				identifiers = ["load"]
+				wrapperAst = parse(`${wrapper}(/*options here*/).wrap`)
+				importFrom = "@inlang/sdk-js/adapter-sveltekit/shared"
+				break
+			case "initRequestHandlerWrapper":
+				identifiers = ["GET", "POST", "PUT", "PATCH", "DELETE", "OPTIONS"]
+				wrapperAst = parse(`${wrapper}(/*options here*/).wrap`)
+				importFrom = "@inlang/sdk-js/adapter-sveltekit/server"
+				break
+			case "initActionWrapper":
+				identifiers = ["actions"]
+				wrapperAst = parse(`${wrapper}(/*options here*/).wrap`)
+				importFrom = "@inlang/sdk-js/adapter-sveltekit/server"
+				break
+			default:
+				break
+		}
+		// Wrap the specified functions
+		const wasWrapped = definitions(ast, ...identifiers)
+			.wrap(wrapperAst)
+			.successful()
+		if (wasWrapped) imports(ast, importFrom).add(wrapper)
+
+		// Add imports
+	})
+	const aliases = imports(ast, "@inlang/sdk-js").getAliases()
+	// Ensure that function parameters are correct
+	contexts(ast, ...aliases.map(({ aliasN }) => aliasN))
+		.isFunction()
+		.merge(
+			parse(
+				`function(_, {${aliases
+					.map(({ exportN, aliasN }) => `${exportN}:${aliasN}`)
+					.join(",")}}) {}`,
+			),
+		)
+	aliases.forEach(({ exportN, aliasN }) =>
+		// Wrap all occurences of `i`, `language`, etc with `get()`
+		identifiers(ast, aliasN).wrap(parse("get")),
+	)
+}
+
+function replaceSdkImports(ast) {
+	// Prunes the imports, i.e. merge duplicates.
+	// Then remove sdk imports, but save the aliases: import {i as iAlias} ... returns [{exportN: string, aliasN: string}, ...]
+	const aliases = imports(ast, "@inlang/sdk-js").prune().remove().getAliases()
```

Comment:
from reading the code I find it strange that you call `remove` and afterwards `getAliases()`. I would split those things into two statements.